### PR TITLE
Fix on code quality based on Pylint

### PR DESCRIPTION
**Pull request by SainTurDaY27**
Hello everyone!!
I have fix some code to high quality using Pylint on users, actions directory.
The rest of code that not fixed yet. @Siraphop4Nene will complete it after this.
In this PR, I have added .pylintrc for ignore Pylint check cases & change limit of some format too.

Please review my work.
Thank you!

### CODE REVIEW

**Comment by SainTurDaY27** [comment]
After ask for PR, CodeFactor told me that I need to ignore migration files on .pylintrc.
I will fix it!

**Review by Siraphop4Nene — APPROVED** [review]

**Review by Siraphop4Nene — APPROVED** [review]

**Comment by SainTurDaY27** [comment]
Now I have fix .pylintrc to ignore files that contains migration data.

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        wrong-import-order,
+        ungrouped-imports,
+        consider-using-f-string,
+        import-self,
+        unspecified-encoding,
+        consider-using-with,
+        too-many-locals,
+        broad-except,
+        reimported,
+        consider-using-in,
```
Comment (pontakornth) [inline .pylintrc:40]: I think this is important. Using `in` will make conditions more concise.

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# --disable=W"
+disable=undefined-variable,
+        wildcard-import,
+        relative-beyond-top-level,
+        import-outside-toplevel,
+        unused-import,
+        no-member,
+        too-few-public-methods,
+        unused-argument,
+        wrong-import-order,
```
Comment (pontakornth) [inline .pylintrc:31]: We can use autoformatter on this one.

File: .pylintrc [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+# disable everything first and then reenable specific checks. For example, if
+# you want to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use"--disable=all --enable=classes
+# --disable=W"
+disable=undefined-variable,
+        wildcard-import,
+        relative-beyond-top-level,
+        import-outside-toplevel,
+        unused-import,
```
Comment (pontakornth) [inline .pylintrc:27]: We can use autoformatter to remove unused import.

**Review by pontakornth — COMMENTED** [review]
I think some changes may be discussed.

**Review by HelloYeew — APPROVED** [review]
Looking closely and it's looking good!

**Review by pontakornth — APPROVED** [review]

**Review by some-random-idiot — APPROVED** [review]

**Comment by HelloYeew** [comment]
Let's try it!

**Pull request merged**